build/pubspec.yaml

```diff
@@ -24,3 +24,15 @@ dev_dependencies:
   build_vm_compilers: ^0.1.0
   pedantic: ^1.0.0
   test: ^1.2.0
+
+dependency_overrides:
+  build_daemon:
+    path: ../build_daemon
+  build_runner:
```

Comment:
You can likely get rid of at least some of these now that I published all the packages

Reply:
Done.